### PR TITLE
Storybook > Pointers

### PR DESCRIPTION
**Pull request by edderrd**
Update default pointers to retro styles

### CODE REVIEW

**Comment by edderrd** [comment]
[Storybook > Pointers](https://trello.com/c/IiO8vzmR/15-storybook-pointers)

File: util/theme.js [hunk 1]
```diff
@@ -7,6 +7,19 @@ const offsetDotSvgUrl =
 const offsetRoundedDotSvgUrl =
   'url(\'data:image/svg+xml;utf8,<svg height="15" viewBox="0 0 10 15" width="10" xmlns="http://www.w3.org/2000/svg"><g fill="%23000" fill-rule="evenodd"><rect height="5" rx="1" width="5"/><rect height="5" rx="1" width="5" x="5" y="7.5"/></g></svg>\')'
 
+export const defaultPointerUrl =
+  // eslint-disable-next-line prettier/prettier
```
Comment (mckelvey) [inline util/theme.js:11]: Curious... what was prettier complaining about here?

File: util/theme.js [hunk 2]
```diff
@@ -7,6 +7,19 @@ const offsetDotSvgUrl =
...
+  'url(\'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACwAAABACAYAAABhspUgAAAAzklEQVRoge3Zyw6DIBRFUWj6/79Mp+3R8Ige4dK9Z8aADFauMeZSSkn1cuP+o71WOkxPHNjdW/fP+ZfsifGppiHhLr5hbTXTkHC3n2FttmlIuNvfsPa0aUi4+z/Dmts0JNxhWLvbNCTcYbjVVdOQcIfh0UZNQ8IdhrX2L5T2Ft8XkHC3v2E1qXO0I77pli7cgQ//mk/e7Yc1cl1dr3XsVw0S7uLP4avG3EHCXfw5fINZ61yGhLuQc3ipOdsKEu5CzmH3M0YfwByeWqwDp5Q+NylFhR7a21kAAAAASUVORK5CYII=\')'
```
Comment (mckelvey) [inline util/theme.js:12]: These seem to be 64x64 images; but should be 32, will you reduce?

**Review by mckelvey — CHANGES_REQUESTED** [review]
@edderrd — The cursors are coming across as 2x size for me and there’s no CSS control for that, you might just have to reduce the actual image sizes.

File: util/theme.js (hunk 1 above)
Comment (edderrd) [reply util/theme.js:11]: yes, eslint and prettier was fighting against each other

File: util/theme.js (hunk 2 above)
Comment (edderrd) [reply util/theme.js]: yeah, not sure why got exported like that, should be fixed now

File: util/theme.js (hunk 1 above)
Comment (mckelvey) [reply util/theme.js:11]: Oh I hate that.

**Review by mckelvey — APPROVED** [review]
👍

**Pull request merged**